source/clear-linux/tutorials/azure.rst

```diff
@@ -1,125 +1,481 @@
+your user name:
+
+.. code-block:: console
+
+   SSH key files '/home/$USER/.ssh/id_rsa' and '/home/$USER/.ssh/id_rsa.pub' have been generated under ~/.ssh to allow SSH access to the VM. If using machines without permanent storage, back up your keys to a safe location.
+
+   running...
+
+   {
+     "fqdns": "",
+     "id": "/subscriptions/{unique-id}/resourceGroups/ClearResourceGroup/providers/Microsoft.Compute/virtualMachines/ClearVM",
+     "location": "westus",
+     "macAddress": "00-0D-3A-37-C7-59",
+     "powerState": "VM running",
+     "privateIpAddress": "10.0.0.4",
+     "publicIpAddress": "13.91.4.245",
+     "resourceGroup": "ClearResourceGroup",
+     "zones": ""
+   }
+
+Take note of the public IP address from your output. To login into the new
+|CL| VM, run the :command:`ssh` command with the public IP address listed as
+shown:
+
+.. code-block:: console
+
+   ssh 13.91.4.245
+   
+You may see the following message about the authenticity of the host. If this
+appears, type `yes` to proceed connecting to your new |CL| VM.
+
+.. code-block:: console
+
+   The authenticity of host '13.91.4.245 (13.91.4.245)' can't be established.
+   RSA key fingerprint is SHA256:{unique-number}.
+   Are you sure you want to continue connecting (yes/no)? yes
+   Warning: Permanently added '13.91.4.245' (RSA) to the list of known hosts.
+
+   USER@ClearVM ~ $
+
+You are now logged into your new |CL| VM as USER, where USER is your user
+name. Let's check to see which software bundles have been included with this
+image by running the :command:`swupd` command:
+
+.. code-block:: console
+
+   USER@ClearVM ~ $ sudo swupd bundle-list
+   swupd-client bundle list 3.14.1
+      Copyright (C) 2012-2017 Intel Corporation
+
+   bootloader
+   editors
+   kernel-hyperv
+   network-basic
+   openssh-server
+   os-cloudguest-azure
+   os-core
+   os-core-update
+   perl-basic
+   python-basic
+   python3-basic
+   storage-utils
+   sysadmin-basic
+   Current OS version: 19600
+   USER@ClearVM ~ $ 
+
+When you are finished using your new |CL| VM, type :command:`exit` to close
+the terminal and logout.
+
+Stop and deallocate the Clear Linux VM
+**************************************
+
+When you are finished using your new |CL| instance, you need to stop the VM
+and deallocate the resources to stop incurring charges for this instance. At
+your command prompt, enter the :command:`az vm stop...` command as follows:
+
+.. code-block:: console
+
+   az vm stop --resource-group ClearResourceGroup --name ClearVM
+
+This will stop the VM and then output text similar to what is shown below:
+
+.. code-block:: console
+
+   {
+     "endTime": "2017-12-13T23:04:02.346676+00:00",
+     "error": null,
+     "name": "{unique-name}",
+     "startTime": "2017-12-13T23:03:59.018536+00:00",
+     "status": "Succeeded"
+   }
+
+
+Once the VM has stopped, deallocate the VM resources to stop incurring charges
+for the |CL| instance. Enter the following command:
```

Comment:
> + Once the VM has stopped, deallocate the VM resources to stop incurring charges
> +for the |CL| instance.

I'm assuming deallocating means that the VM is destroyed.  Maybe a warning note?

Reply:
The VM is stopped and saved, as are any resources.